src/core/wb.js

```diff
@@ -571,14 +571,29 @@ Modernizr.load( [
 					// Start initialization
 					wb.init( document, componentName, selector );
 
+					// Disable MathJax's context menu to more closely mimic native MathML implementations
+					window.MathJax = {
+						options: {
+							enableMenu: false
+						}
+					};
+
 					// Load the MathML dependency. Since the polyfill is only loaded
 					// when !Modernizr.mathml, we can skip the test here.
 					Modernizr.load( [ {
-						load: "timeout=500!https://cdn.jsdelivr.net/npm/mathjax@WET_BOEW_VERSION_MATHJAX/MathJax.js?config=Accessible",
+
+						// Load ES6 polyfill (for IE11) and MathJax from CDNs
+						load: [
+							"timeout=500!https://polyfill.io/v3/polyfill.min.js?features=es6",
+							"timeout=500!https://cdn.jsdelivr.net/npm/mathjax@WET_BOEW_VERSION_MATHJAX/es5/mml-chtml.js"
+						],
 						complete: function() {
+
+							// Fall back on a local copy MathJax if CDN loading fails
+							// Note: Won't work with IE11 in isolated networks (ES6 polyfill has no local fallback)
 							Modernizr.load( [ {
-								test: window.MathJax === undefined,
-								yep: "mthjx!MathJax.js?config=Accessible"
+								test: window.MathJax.startup === undefined,
```

Comment:
You might need to use the trick like "IsReady" and implement a little delay before to check again if it is loaded or not.